src/combobox/type.ts

```diff
@@ -18,9 +18,10 @@ export declare type ComboboxProps = {
   error?: string;
   id?: string;
   label?: string;
-  value?: string;
   disabled?: boolean;
+  placeholder?: string;
   requiredIcon?: boolean;
+  value?: string;
```

Comment:
Please sort the properties by group.
```suggestion
  placeholder?: string;
  value?: string;
  disabled?: boolean;
  requiredIcon?: boolean;
```